assets/scss/02-molecules/_pagination.scss

```diff
@@ -29,8 +30,8 @@
       margin: 0 .1em;
     }
 
-    &:disabled {
-      cursor: default;
+    &.disabled {

```

Comment:
why change disable from using attribution to class?

Reply:
@clair0917 Can you tell us why this change was done?